app/javascript/src/components/Dashboard/Notes/NoteList/index.jsx

```diff
@@ -0,0 +1,45 @@
+import React, { useState } from "react";
+
+import Note from "./Note";
+
+import DeleteAlert from "../DeleteAlert";
+import EditNotePane from "../Pane/Edit";
+
+const NoteList = ({ notes = [], fetchNotes }) => {
+  const [showEditNote, setShowEditNote] = useState(false);
+  const [selectedNote, setSelectedNote] = useState({});
+  const [showDeleteAlert, setShowDeleteAlert] = useState(false);
+  const [selectedDeleteNoteIds, setSelectedDeleteNoteIds] = useState([]);
+  return (
+    <>
+      <div>
+        {notes.map((note, idx) => (
+          <Note
+            key={idx}
+            note={note}
+            setSelectedDeleteNoteIds={setSelectedDeleteNoteIds}
```

Comment:
This name can be made better:
```
setNoteIdsToBeDeleted
```

Reply:
Ok thanks Yedhin. I will make sure that names are more expressive.